lib/page/resetpassword/reset_password_page.dart

```diff
@@ -0,0 +1,157 @@
+import 'package:another_flushbar/flushbar.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:survey/constants.dart';
+import 'package:survey/di/di.dart';
+import 'package:survey/gen/assets.gen.dart';
+import 'package:survey/gen/colors.gen.dart';
+import 'package:survey/page/resetpassword/reset_password_state.dart';
+import 'package:survey/page/resetpassword/reset_password_view_model.dart';
+import 'package:survey/resource/dimens.dart';
+import 'package:survey/usecase/reset_password_use_case.dart';
+import 'package:survey/widget/app_bar_back_button_widget.dart';
+import 'package:survey/widget/loading_indicator_widget.dart';
+import 'package:survey/widget/onboarding_background_widget.dart';
+import 'package:survey/widget/rounded_button_widget.dart';
+import 'package:survey/widget/text_input_widget.dart';
+
+final resetPasswordViewModelProvider = StateNotifierProvider.autoDispose<
+    ResetPasswordViewModel, ResetPasswordState>((ref) {
+  return ResetPasswordViewModel(getIt.get<ResetPasswordUseCase>());
+});
+
+class ResetPasswordPage extends ConsumerStatefulWidget {
```

Comment:
Could you please rename the package in PR #76 instead, I would like to see the diff on this page for reviewing 🙏